nym-vpn-core/crates/nym-statistics/src/config.rs

```diff
@@ -0,0 +1,70 @@
+// Copyright 2025 - Nym Technologies SA <contact@nymtech.net>
+// SPDX-License-Identifier: GPL-3.0-only
+
+use nym_sdk::UserAgent;
+use url::Url;
+
+const STATS_REPORT_MAX_DELAY_SECS: u64 = 60;
+
+#[derive(Debug, Clone)]
+pub struct StatisticsControllerConfig {
+    /// Is stats reporting enabled
+    pub(crate) enabled: bool,
+
+    /// Address of the stats collector. If this is none, no reporting will happen, regardless of `enabled`
+    pub stats_collector_url: Option<Url>,
+
+    pub user_agent: UserAgent,
+
+    // Allow sending reports even with a not connected tunnel
```

Comment:
Do you need some facilities to poke holes in firewall to arrange that or how does that work?